src/models/outbound/pispOTPValidate.interface.ts

```diff
@@ -21,22 +21,40 @@
  * Gates Foundation
  - Name Surname <name.surname@gatesfoundation.com>
 
- - Kevin Leyow <kevin.leyow@modusbox.com>
+ - Kevin Leyow - kevin.leyow@modusbox.com
  --------------
  ******/
+import {
+  ControlledStateMachine,
+  PersistentModelConfig, StateData
+} from '~/models/persistent.model'
+import { Method } from 'javascript-state-machine'
+import { ThirdpartyRequests } from '@mojaloop/sdk-standard-components';
+import {
+  thirdparty as tpAPI
+} from '@mojaloop/api-snippets'
+import { PubSub } from '~/shared/pub-sub'
+export interface PISPOTPValidateStateMachine extends ControlledStateMachine {
+  validateOTP: Method
+  onValidateOTP: Method
+}
+
+export interface PISPOTPValidateModelConfig extends PersistentModelConfig {
+  pubSub: PubSub
+  thirdpartyRequests: ThirdpartyRequests
+  requestProcessingTimeoutSeconds: number
+}
+
+export interface PISPOTPValidateData extends StateData {
+  currentState: 'start' | 'OTPIsValid' | 'errored',
```

Comment:
when using generic param for StateData - you are not forced to redefine currentState and use the source of truth delivered by autogenerated interfaces/types